src/media_tools/media_import.c

```diff
@@ -2326,6 +2338,8 @@ GF_Err gf_import_isomedia(GF_MediaImporter *import)
 
 		gf_isom_copy_sample_info(import->dest, track, import->orig, track_in, i+1);
 
+		gf_isom_set_sample_roll_group(import->dest, track, i+1, import->audio_roll);
```

Comment:
1- could you also enclose this one in a if (import->audio_roll) { } ?
2- maybe for isobmff import we want to reset the roll to 0 if the source track already has roll info ?

Reply:
1 - done
2 - done.